pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "splitapiclient"
-version = "3.5.4"
+version = "3.6.0.rc.1"
```

Comment:
@chillaq I believe this should be 3.6.0-rc.1 to fit the SemVer format? https://semver.org/spec/v2.0.0-rc.2.html#spec-item-9

Reply:
it actually does not matter, the pip engine removes "-" or "." before the rc, so resulted version is always "3.6.0rc1"